kf_hungarian_tracker/kf_hungarian_tracker/kf_hungarian_node.py

```diff
@@ -167,11 +165,12 @@ def callback(self, msg):
             marker_list = []
             # add current active obstacles
             for obs in filtered_obstacle_list:
-                (r, g, b) = colorsys.hsv_to_rgb(obs.msg.id * 30. % 360 / 360., 1., 1.) # encode id with rgb color
+                (r, g, b) = colorsys.hsv_to_rgb((obs.msg.id.int >> 64) * 30. % 360 / 360., 1., 1.) # encode id with rgb color
```

Comment:
When I run this in ipython a few times, its always gives me out the same 3 values: 0.25, 0.916, and 0.58.

```
(uuid.uuid1().int >> 64) * 30 % 360 / 360
```

I don't think this works.